appmanifest/shortcuts-member/shortcuts-member.webmanifest

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Shortcut test",
+    "scope": "/",
+    "shortcuts": [{
+        "name": "pass",
+        "short_name": "",
+        "description": "",
+        "url": "shortcuts-member-manual.html",
```

Comment:
I think we should have a "shortcut_pass.html" file instead. We could then use that throughout the tests to make sure the shortcut is landing in the right place. 

The file could just be:

```HTML 
<html>
<meta charset="utf-8">
<body style="background: green">
<h1>Pass</h1>
```

Similarly, we could have a fail version of the above with a red background.